wfe/context.go

```diff
@@ -0,0 +1,85 @@
+package wfe
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/core"
+	blog "github.com/letsencrypt/boulder/log"
+)
+
+type requestEvent struct {
+	ID           string    `json:",omitempty"`
+	RealIP       string    `json:",omitempty"`
+	ClientAddr   string    `json:",omitempty"`
+	Endpoint     string    `json:",omitempty"`
+	Method       string    `json:",omitempty"`
+	RequestTime  time.Time `json:",omitempty"`
+	ResponseTime time.Time `json:",omitempty"`
+	Errors       []string
+	Requester    int64           `json:",omitempty"`
+	Contacts     []*core.AcmeURL `json:",omitempty"`
```

Comment:
What's the utility of logging this?  Since we have the registration ID, we can look up current contacts.